source

```diff
@@ -105652,15 +105751,21 @@ document.body.appendChild(text);
    <dt>A start tag whose tag name is one of: "head", "noscript"</dt>
    <dt>Any other end tag</dt>
    <dd>
-    <p><span>Parse error</span>. Ignore the token.</p>
+    <p>This is an <span
```

Comment:
We shouldn't report this error for `A start tag whose tag name is one of: "head", "noscript"` case. Instead we need to report `misplaced-start-tag-for-head-element` error in case of "head" and new `nested-noscript-in-head` error for "noscript".